src/aurora/xmlfixer.cpp

```diff
@@ -186,10 +191,39 @@ Common::UString XMLFixer::fixXMLValue(const Common::UString value) {
 	return "\"" + line + "\"" + tag;
 }
 
+/**
+ * Address special issues found in specific NWN2 XML files
+ * by looking for exact matches to the problematic value
+ * strings then correcting the value on a match.
+ */
+bool XMLFixer::isFixSpecialCase(Common::UString *value) {
+	bool isFix = false;
+	const int rows = 2;
+	const Common::UString swap[rows][2] = {
+		{ "truefontfamily",
+		  "\"true\" fontfamily" },	// examine.xml
+		{ "Character\"fontfamily",
+		  "\"Character\" fontfamily" },	// multiplayer_downloadx2.xml
+	};
+
+	// Loop through the array
+	for (int i = 0; i < rows; i++) {

```

Comment:
You should be able to just define it as `swap[2][2]`, and then use the `ARRAYSIZE()` macro to get the number of rows, no?